u9/containers/Services/Services.tsx

```diff
@@ -0,0 +1,216 @@
+        image: "./images/services-vr-mr-ar.png",
+      },
+      {
+        title: "Metaverse",
+        description:
+          "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus gravida congue velit sit amet lacinia. Nam diam enim, tincidunt ut sapien at, porta tempus nisl. Curabitur finibus dui lectus, ut mollis libero pharetra id. In id accumsan enim, non aliquet massa. Sed euismod purus ut luctus faucibus. Sed varius orci magna, et porta ligula porta id. Fusce sit amet interdum neque.",
+        image: "./images/services-metaverse.png",
+      },
+      {
+        title: "Desktop Appraisal",
+        description:
+          "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus gravida congue velit sit amet lacinia. Nam diam enim, tincidunt ut sapien at, porta tempus nisl. Curabitur finibus dui lectus, ut mollis libero pharetra id. In id accumsan enim, non aliquet massa. Sed euismod purus ut luctus faucibus. Sed varius orci magna, et porta ligula porta id. Fusce sit amet interdum neque.",
+        image: "./images/services-desktop-appraisal.png",
+      },
+      {
+        title: "IoT",
+        description:
+          "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus gravida congue velit sit amet lacinia. Nam diam enim, tincidunt ut sapien at, porta tempus nisl. Curabitur finibus dui lectus, ut mollis libero pharetra id. In id accumsan enim, non aliquet massa. Sed euismod purus ut luctus faucibus. Sed varius orci magna, et porta ligula porta id. Fusce sit amet interdum neque.",
+        image: "./images/services-iot.png",
+      },
+      {
+        title: "Mobile App & Web Tools",
+        description:
+          "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus gravida congue velit sit amet lacinia. Nam diam enim, tincidunt ut sapien at, porta tempus nisl. Curabitur finibus dui lectus, ut mollis libero pharetra id. In id accumsan enim, non aliquet massa. Sed euismod purus ut luctus faucibus. Sed varius orci magna, et porta ligula porta id. Fusce sit amet interdum neque.",
+        image: "./images/services-mobile-app-web-tools.png",
+      },
+    ],
+    phases: [
+      {
+        phase: "Phase 1",
+        title: "Discovery",
+        description:
+          "Nullam vulputate gravida enim, et tincidunt augue condimentum at. Nullam ultricies felis nisl, vitae pellen venenatis quis. Vitae vestibulum arcu, a semper metus nullam placerat.",
+        image: "./images/services-phase-1.png",
+      },
+      {
+        phase: "Phase 2",
+        title: "Production of MVP",
+        description:
+          "Nullam vulputate gravida enim, et tincidunt augue condimentum at. Nullam ultricies felis nisl, vitae pellen venenatis quis. Vitae vestibulum arcu, a semper metus nullam placerat.",
+        image: "./images/services-phase-2.png",
+      },
+      {
+        phase: "Phase 3",
```

Comment:
phase 3 and 4 come out as grey, both text and images. there's a "non greyed" version of such images in figma, can we please replace them, and have the text the same color of phase 1 and 2?